services/order.py

```diff
@@ -0,0 +1,28 @@
+from django.db import transaction
+from db.models import Order, Ticket, User
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
```

Comment:
The type annotation `list[dict]` is not compatible with Python versions below 3.9. Consider using `List[Dict]` from the `typing` module for better compatibility.